### ISSUE TITLE
Multifaced Cards - automatically update? 

### ISSUE THREAD

**Issue by gfrt0** [cderickson/MTGO-Tracker#31]
Hi, first of all thanks for the great tool. I am still learning how to use it, so apologies if any of this is trivial. 

Inspecting the MULTIFACED_CARDS.txt file shows it (1) is outdated and (2) contains Arena-only cards. 

For (1), would it not be better to have `Update Auxiliary Files` pull directly from Scryfall instead of Github?

For both (1) and (2), the following calls return the multifaced cards available on MTGO as of this issue: 

```
* SPLIT CARDS
http://api.scryfall.com/cards/search?q=is:split+in:mtgo&format=csv&page=1

* TRANSFORM CARDS
http://api.scryfall.com/cards/search?q=is:transform+in:mtgo&format=csv&page=1
http://api.scryfall.com/cards/search?q=is:transform+in:mtgo&format=csv&page=2

* DFCs
http://api.scryfall.com/cards/search?q=is:dfc+in:mtgo&format=csv&page=1
http://api.scryfall.com/cards/search?q=is:dfc+in:mtgo&format=csv&page=2
http://api.scryfall.com/cards/search?q=is:dfc+in:mtgo&format=csv&page=3

* MDFCs
http://api.scryfall.com/cards/search?q=is:mdfc+in:mtgo&format=csv&page=1

* ADVENTURE CARDS
http://api.scryfall.com/cards/search?q=is:adventure+in:mtgo&format=csv&page=1
```

(I am outputting to CSV because it was easier for me to copy-paste to .txt).

**Comment by cderickson** [comment]
Hey, thanks for trying out to tool, I hope you find it useful!

I have a script that generates MULTIFACED_CARDS.txt using the Scryfall API. Having the tool pull directly from Scryfall instead of relying on me to update it probably makes more sense, I'll put in on my backlog of changes to make. I think when I decided to do it this way, I wasn't expecting flip/split cards to be released so regularly.

**Comment by gfrt0** [comment]
Yeah I imagined that was the pipeline, the suggestion is intended to free you up of having to update it every new release (i.e. every 2 weeks, nowadays).
